jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/include/baxter-interface.l

```diff
@@ -0,0 +1,287 @@
+;;
```

Comment:
@pazeshun why this file is necessary? Is that impossible to reuse baxter-interface.l in jsk_2015_05_baxter_apc or baxtereus?